pkgs/development/tools/analysis/banana-vera/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, python310
+, boost
+, tcl
+}:
+
+stdenv.mkDerivation {
+  pname = "banana-vera";
+  version = "1.3.0-python3.10";
+
+  src = fetchFromGitHub {
+    owner = "Epitech";
+    repo = "banana-vera";
+    rev = "3a7d18d6249cd1790cc63fb9018e0914462ec219";
+    sha256 = "sha256-1nAKhUltQS1301JNrr0PQQrrf2W9Hj5gk1nbUhN4cXw=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [
+    python310
+    (boost.override {
+      enablePython = true;
+      python = python310;
+    })
```

```suggestion
    python310.pkgs.boost
```
`boost` on python-packages already has python-enabled `include/`.